.eslintignore

```diff
@@ -1,3 +1,3 @@
-bin/*
-dist/*
-test/*
+bin/
+examples/
```

Comment:
I just wanted to lint `test/` folder, so `"lint": "eslint src/"` is now `"lint": "eslint ."` and we need to ignore some folders in .eslintignore, like `examples/`